dig.go

```diff
@@ -309,6 +342,12 @@ func (c *Container) getProviders(k key) []provider {
 	return providers
 }
 
+// Invoker return a function to run when calling function provided to Provide or Invoke. Used for
+// running container in dry mode.
+func (c *Container) Invoker() Invoker {
```

Comment:
should unexport. can rename field to `invokeFn` to avoid conflict with method name.